### DISCUSSION TITLE
Query parameter _search

### DISCUSSION THREAD

**Discussion by chgo-invit** [FirelyTeam/firely-net-sdk#2603] [Q&A] [answered]
Disclaimer: I'm quite new to both FHIR and the SDK.

I'm trying to query a FHIR-server that requires the search the be on the form: [server]/fhir/Patient/_search?given=test

I suspect that the "_search"-part is not being generated by the FhirClient. But can't really verify this since I can't inspect the request. It works just fine in Postman.

Is there any way to include "_search" or should I do something completely different?

**Comment by mbaltus** [comment, marked as answer]
The _search can only be used if the search is performed as a POST. See the syntax here: http://hl7.org/fhir/http.html#search.
I assume you are currently using the regular `Search` method of the FhirClient. This will do a GET and not include the _search - as is correct according to the specification. If you need to include _search, use the `SearchUsingPost` method, because then the rules for a POST search will be followed.
